src/components/organisms/PublicHeader.vue

```diff
@@ -0,0 +1,34 @@
+<script setup>
+import useLayout from '~/composables/layout'
+import { APP_TITLE } from '~/constants'
+import DcBtn from '../atoms/DcBtn.vue';
+
+const { toggleCartDrawer, toggleMenuDrawer } = useLayout()
+</script>
+
+<template>
+  <QHeader elevated class="bg-dark text-white" style="height: 80px" height-hint="80">
+    <QToolbar class="full-height">
+      <QToolbarTitle>
+        <QAvatar>
+          <img src="/favicon.svg">
+        </QAvatar>
+        {{ APP_TITLE }}
+      </QToolbarTitle>
+
+      <div class="q-gutter-x-md" v-if="$q.screen.gt.sm" >
+        <DcBtn icon="i-carbon-user" label="Entrar" class="public-header__login-btn"></DcBtn>
+        <QBtn dense flat round icon="i-carbon-shopping-cart" @click="toggleCartDrawer" />
+      </div>
+
+      <QBtn
+        v-if="$q.screen.lt.md"
```

Comment:
I'm new to Quasar but I'm assuming this $q is used by Quasar. Can you explain me exactly what this property mean?

Reply:
Yeah, `$q` is an global variable that we can access in our template without importing, so we can have access to the quasar api, with configuratios, plugins, and computed properties like (`dark mode`, `loading bar`, `lang(already built in i18n), notify(Notify plugin function to create notifications), screen(with this we can access the screen computed property with x, y value and conditions based on our breakpoints`,